odabot/cli.py

```diff
@@ -63,6 +67,24 @@ def send_email(_to, subject, text):
     except Exception as e:
         logger.error('Exception while sending email: %s', e)    
 
+
+def set_commit_state(proj_id, commit_sha, name, state, target_url=None, description=None):
+    gitlab_api_token = os.getenv("GITLAB_API_TOKEN")
+    if gitlab_api_token is None:
+        logger.warning("Gitlab api token not set. Skipping commit state update.")
+        return
+    params = {'name': name, 'state': state}
```

Comment:
Since they are all clustered under "External", maybe it's good to show something like "MMODA: build" else it's only obvious at hover

Reply:
And the hover could reveal more info, like: 
- container image (when build) and url to it if its pushed externally
- test stage to show some self-test information
- frontend url, I guess it's anticipated but not showing?

Reply:
Also, if graph record is created, would be really nice to mention it somewhere, put a reference to it.
Maybe put another step like "registering in ODA KG" :)